src/components/Tool/Appeal/Flow/ContactFlow.tsx

```diff
@@ -167,6 +257,43 @@ export const ContactFlow: React.FC<ContactFlowProps> = ({
           ))}
         </Box>
       </DndProvider>
+
+      {addExcludedContactModalOpen && contact && (
+        <DynamicAddExcludedContactModal
+          contactIds={[contact.id]}
+          handleClose={() => setAddExcludedContactModalOpen(false)}
+        />
+      )}
+
+      {deleteAppealContactModalOpen && contact?.id && (
+        <DynamicDeleteAppealContactModal
+          contactId={contact.id}
+          handleClose={() => setDeleteAppealContactModalOpen(false)}
+        />
+      )}
+
+      {pledgeModalOpen && contact && (
+        <DynamicPledgeModal
+          contact={contact}
+          pledge={contact.pledge}
+          handleClose={() => setPledgeModalOpen(false)}
+        />
+      )}
+
+      {deletePledgeModalOpen && contact?.pledge && (
```

Comment:
Just for consistency

```suggestion
      {deleteAppealContactModalOpen && contact && (
        <DynamicDeleteAppealContactModal
          contactId={contact.id}
          handleClose={() => setDeleteAppealContactModalOpen(false)}
        />
      )}

      {pledgeModalOpen && contact && (
        <DynamicPledgeModal
          contact={contact}
          pledge={contact.pledge}
          handleClose={() => setPledgeModalOpen(false)}
        />
      )}

      {deletePledgeModalOpen && contact && (
```